ansible/roles/azure_ansible_execution_env/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
Can you fill this out?

Reply:
I can indeed! I'll do a little more clean up to.

Reply:
Done below